### ISSUE TITLE
Invalid references in twisted.names

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8517]
|[<img alt="acabhishek942's avatar" src="https://avatars.githubusercontent.com/u/7778186?s=50" width="50" height="50">](https://github.com/acabhishek942)| @acabhishek942 reported|
|-|-|
|Trac ID|trac#8517|
|Type|enhancement|
|Created|2016-06-27 18:51:00Z|

Following are the invalid references in twisted.names obtained by checking out the trunk and running

```
./bin/admin/build-apidocs . apidocs

```

```
twisted.names.client.Resolver.filterAnswers: invalid ref to dns.OK
twisted.names.dns.Message.lookupRecordType: invalid ref to A
twisted.names.dns.randomSource: invalid ref to randbytes.secureRandom
twisted.names.error.DNSFormatError: invalid ref to twisted.names.dns.EFORMAT
twisted.names.error.DNSNameError: invalid ref to twisted.names.dns.ENAME
twisted.names.error.DNSNotImplementedError: invalid ref to twisted.names.dns.ENOTIMP
twisted.names.error.DNSQueryRefusedError: invalid ref to twisted.names.dns.EREFUSED
twisted.names.error.DNSServerError: invalid ref to twisted.names.dns.ESERVER
twisted.names.root.Resolver._discoverAuthority: invalid ref to RRHeaders
twisted.names.root.Resolver._discoveredAuthority: invalid ref to RRHeaders
twisted.names.root.Resolver._query: invalid ref to RRHeaders
```

<details><summary>Searchable metadata</summary>

```
trac-id__8517 8517
type__enhancement enhancement
reporter__acabhishek942 acabhishek942
priority__normal normal
milestone__None None
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1467053460819288 1467053460819288
changetime__1474198049187875 1474198049187875
version__None None
owner__rodrigc rodrigc

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="acabhishek942's avatar" src="https://avatars.githubusercontent.com/u/7778186?s=50" width="50" height="50">](https://github.com/acabhishek942)<a name="note_1"></a>|@acabhishek942 commented|
|-|-|

See: https://github.com/twisted/twisted/pull/255

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_2"></a>|@rodrigc set owner to @rodrigc|
|-|-|

Looks good

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_3"></a>|GitHub set status to `closed`|
|-|-|

In changeset 46ca5f66bf3f3ab59c1031ab9ba182a3c68e9184
```
#!CommitTicketReference repository="" revision="46ca5f66bf3f3ab59c1031ab9ba182a3c68e9184"
Merge 8517-acabhishek942-doc: Fix invalid references in twisted.names

Author: acabhishek942
Reviewer: rodrigc
Fixes: #8517
```

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_4"></a>|@hawkowl commented|
|-|-|

[mass edit] Removing review from closed tickets.
